tests/feature/test_gherkin_terminal_reporter.py

```diff
@@ -344,3 +345,22 @@ def output_output_must_contain_parameters_values(test_execution, gherkin_scenari
     ghe.stdout.fnmatch_lines('*When I eat {eat} cucumbers'.format(**gherkin_scenario_outline))
     ghe.stdout.fnmatch_lines('*Then I should have {left} cucumbers'.format(**gherkin_scenario_outline))
     ghe.stdout.fnmatch_lines('*PASSED')
+
+
+@pytest.mark.parametrize(
+    'feature_file, py_file, name', [
+        ('./steps/unicode.feature', './steps/test_unicode.py', 'test_steps_in_feature_file_have_unicode')
+    ]
+)
+def test_scenario_in_expanded_mode(testdir, test_execution, feature_file, py_file, name):
+    prepare_feature_and_py_files(testdir, feature_file, py_file)
```

Comment:
you can use `testdir.copy_example()` instead. Also, I think the test parametrization is a little bit useless, you can just hardcode the file names in here

Reply:
Correct me if I am wrong but `testdir.copy_example()` has been added to `pytest==3.7.0` https://docs.pytest.org/en/latest/changelog.html#pytest-3-7-0-2018-07-30
So we can't use it as we want our tests to pass on versions lower than 3.7.0.

Regarding parametrization. I agree that it may look useless here. However I did it this way as this is the place which I want to update in the other opened pull request - simply I want to run another test in expanded mode that is currently failing.

Reply:
Right, I didn't see that. At this point, I am ok with requiring pytest 3.7.0 to be honest and be done with keeping compatibility with older versions (just like the problems we had with node.get_marker).
@bubenkoff, what do you think?

Reply:
Is that not something that we should do in separate pull request? ;-)

Reply:
Sure